src/opm/parser/eclipse/Parser/ParserItem.cpp

```diff
@@ -50,14 +51,26 @@ template<> const std::string& default_value< std::string >() {
     return value;
 }
 
-type_tag get_type_json( const std::string& str ) {
+type_tag get_data_type_json( const std::string& str ) {
     if( str == "INT" )       return type_tag::integer;
     if( str == "DOUBLE" )    return type_tag::fdouble;
     if( str == "STRING" )    return type_tag::string;
     if( str == "RAW_STRING") return type_tag::string;
+    if( str == "UDA")        return type_tag::fdouble;
     throw std::invalid_argument( str + " cannot be converted to enum 'tag'" );
 }
 
+/*
+  For very small numbers std::to_string() will just return the string "0.000000"
+*/
+std::string as_string(double value) {
+    if (std::fabs(value) < 1e-4) {
```

Comment:
You need `#include <cmath>` for `std::fabs()`.

Reply:
done